Doc/whatsnew/3.14.rst

```diff
@@ -479,6 +479,11 @@ New Features
   an interned string and deallocate it during module shutdown.
   (Contribued by Eddie Elizondo in :gh:`113601`.)
 
+* Add :c:func:`PyLong_IsPositive`, :c:func:`PyLong_IsNegative`
+  and :c:func:`PyLong_IsZero` for checking if :c:type:`PyLongObject`
+  is positive, negative, or zero.
+  (Contribued by James Roy in :gh:`126061`.)
```

Comment:
Ask Sergey if he wants to be credited as well :)

Reply:
I think you should credit Serhiy as author of the C-API WG proposal.

Reply:
I'd love to do that. Actually, this opportunity was given to me by Sergey :)

Reply:
> I think you should credit Serhiy as author of the C-API WG proposal.

I am not the author of this PR.